includes/object-cache.php

```diff
@@ -1265,6 +1321,22 @@ protected function is_serialized($data, $strict = true)
 
         return false;
     }
+
+    /**
+     * Handle the redis failure gracefully or throw an exception.
+     *
+     * @param \Exception $exception Exception thrown.
+     */
+    private function handle_redis_failure($exception) {
+        // When Redis is unavailable, fall back to the internal back by forcing all groups to be "no redis" groups
```

Comment:
I know this was an existing comment that was simply moved but is `internal back` a typo?

Reply:
Fixed - 141b0f7